workflow-service/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Sebastián Ramírez
```

Comment:
Copyright Rice University?

Reply:
ah good catch. I modeled the code structure in the attribution section of the README  so licensed it as such b/c it was easier. It's not 1 for 1 copy of the cookiecutter tho. When i want to understand how things work i like to actually start from the minimum so each file is actually hand typed. This is my general approach to learning things ... reverse engineering =)

Reply:
I'd almost leave this file out since there is a LICENSE at the root of the repo.